### PR TITLE
M50 Rebase code format fixing.

### PR DESCRIPTION
**Pull request by shaoboyan**
M50 rebase code format fixing.

### CODE REVIEW

**Comment by shaoboyan** [comment]
@hujiajie  PTAL

**Comment by shaoboyan** [comment]
@hujiajie updated. PTAL

File: third_party/WebKit/Source/modules/webcl/WebCLOpenCL.cpp [hunk 1]
```diff
@@ -215,15 +186,40 @@ cl_int (CL_API_CALL *web_clGetGLTextureInfo)(cl_mem, cl_gl_texture_info, size_t,
 // by this spec.
 #define MAP_FUNC_TO_WRAPPER(fn, major, minor) { *(void**)(&fn) = (void*)fn##Impl##major##minor; }
 
-static const char* defaultLibs[] = LIBS;
-static const int defaultLibsAmount = SO_LEN;
+static Vector<String> defaultLibs;
```
Comment (rakuco) [inline third_party/WebKit/Source/modules/webcl/WebCLOpenCL.cpp]: Having a global static non-POD variable like this is usually a bad idea. You probably need to use `DEFINE_STATIC_LOCAL` or something similar.

File: third_party/WebKit/Source/modules/webcl/DOMWindowWebCL.h [hunk 2]
```diff
@@ -6,11 +6,11 @@
 #define DOMWindowWebCL_h
 
 #include "core/frame/DOMWindowProperty.h"
-#include "core/frame/LocalDOMWindow.h"
```
Comment (hujiajie) [inline third_party/WebKit/Source/modules/webcl/DOMWindowWebCL.h]: This line originally exists in `DOMWindowWebCL.cpp`. Though the header has already been indirectly included there, I suggest to add it back for better readability.

File: third_party/WebKit/Source/modules/webcl/WebCLKernelArgInfoProvider.cpp [hunk 3]
```diff
@@ -157,8 +157,8 @@ void WebCLKernelArgInfoProvider::parseAndAppendDeclaration(const String& argumen
 
 String WebCLKernelArgInfoProvider::extractAddressQualifier(Vector<String>& declarationStrVector)
 {
-    static AtomicString* privateUnderlined = new AtomicString("__private", AtomicString::ConstructFromLiteral);
-    static AtomicString* private = new AtomicString("private", AtomicString::ConstructFromLiteral);
+    static AtomicString* privateQualifierUnderlined = new AtomicString("__privateQualifier", AtomicString::ConstructFromLiteral);
+    static AtomicString* privateQualifier = new AtomicString("privateQualifier", AtomicString::ConstructFromLiteral);
```
Comment (hujiajie) [inline third_party/WebKit/Source/modules/webcl/WebCLKernelArgInfoProvider.cpp]: Please try to name all qualifiers (private / global / read_only / write_only / ...) in a consistent way.

File: third_party/WebKit/Source/modules/webcl/WebCLOpenCL.cpp (hunk 1 above)
Comment (shaoboyan) [inline third_party/WebKit/Source/modules/webcl/WebCLOpenCL.cpp]: You are right. I'll investigate a better way.

**Comment by shaoboyan** [comment]
@hujiajie @rakuco updated. PTAL.

**Comment by hujiajie** [comment]
LGTM. Thanks for the hard work.

**Comment by rakuco** [comment]
lgtm too, ping @mrunalk 
Does this conclude the M50 WebCL rebasing work?

**Comment by shaoboyan** [comment]
@rakuco no this is based on the last patch. and @mrunalk pls squash this patch with the last one.

**Comment by mrunalk** [comment]
@shaoboyan, This was broken again on Linux. Here's the error I got on Linux,

```
ninja: Entering directory `out/Release'
[12042/15833] CXX obj/third_party/WebKit/Source/modules/webcl/modules.DOMWindowWebCL.o
...
-c ../../third_party/WebKit/Source/modules/webcl/DOMWindowWebCL.cpp -o obj/third_party/WebKit/Source/modules/webcl/modules.DOMWindowWebCL.o
In file included from ../../third_party/WebKit/Source/modules/webcl/DOMWindowWebCL.cpp:5:
../../third_party/WebKit/Source/modules/webcl/DOMWindowWebCL.h:17:1: error: [blink-gc] Class 'DOMWindowWebCL' contains invalid fields.
class DOMWindowWebCL final : public GarbageCollectedFinalized<DOMWindowWebCL>, public HeapSupplement<LocalDOMWindow>, public DOMWindowProperty {
^
../../third_party/WebKit/Source/modules/webcl/DOMWindowWebCL.h:34:5: note: [blink-gc] Reference pointer field 'm_window' to a GC managed class declared here:
    LocalDOMWindow& m_window;
    ^
1 error generated.

```
I have fixed it with this follow-up commit, https://github.com/crosswalk-project/chromium-crosswalk/commit/58cafaf72d5273ba2c7fd09ec5698b4f26939dbf
Next time please test the build on Linux.

**Comment by shaoboyan** [comment]
@mrunalk  thx for the fixing,

**Pull request merged**